__tests__/app.test.js

```diff
@@ -92,3 +91,89 @@ describe("/api", () => {
       });
   });
 });
+describe("GET /api/articles", () => {
+  test("should return a 200 status code", () => {
+    return request(app).get("/api/articles").expect(200);
+  });
+
+  test("should return an array of article objects with the correct properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .then(({ body }) => {
+        expect(body).toHaveLength(13);
+
+        body.forEach((article) => {
+          expect(article).hasOwnProperty("comment_count");
+          expect(typeof article.comment_count).toBe("number");
+          expect(article).hasOwnProperty("author");
+          expect(typeof article.author).toBe("string");
+          expect(article).hasOwnProperty("title");
+          expect(typeof article.title).toBe("string");
+          expect(article).hasOwnProperty("article_id");
+          expect(typeof article.article_id).toBe("number");
+          expect(article).hasOwnProperty("topic");
+          expect(typeof article.topic).toBe("string");
+          expect(article).hasOwnProperty("created_at");
+          expect(typeof article.created_at).toBe("string");
+          expect(article).hasOwnProperty("votes");
+          expect(typeof article.votes).toBe("number");
+          expect(article).hasOwnProperty("article_img_url");
+          expect(typeof article.article_img_url).toBe("string");
+          expect(article).not.hasOwnProperty("body");
+        });
+      });
+  });
+  test("should be sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .then(({ body }) => {
+        expect(body).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+});
+describe("GET /api/articles/:article_id/comments", () => {
```

Comment:
What would you expect to happen if an existing article had no comments? This would be a good test to add🙂